### ISSUE TITLE
Psuade-Lite Requirement

### ISSUE THREAD

**Issue by boverhof** [CCSI-Toolset/FOQUS#1142]
UQ and OUU require `psuade-lite`
- Need to tighten up how at least the User is informed 
- UQ and OUU maybe shouldn't be selectable if psuade-lite is not found 
- default psuade location is bogus if psuade is not installed
- in many scenarios this behavior causes UQ Run to hang

Reference
- https://github.com/CCSI-Toolset/FOQUS/issues/1141

**Comment by boverhof** [comment]
# psuade lite location specified in two places
```
$ find . -name \*.py  | xargs grep -s 'psuade_project'
./foqus_lib/framework/uq/LocalExecutionModule.py:                "C:/Program Files (x86)/psuade_project %s/bin/psuade.exe"
./foqus_lib/framework/session/session.py:            or "C:/Program Files (x86)/psuade_project 1.7.5/bin/psuade.exe"
```

**Comment by lbianchi-lbl** [comment]
#1085 would indirectly address the issue of the PSUADE requirement: under the proposed design, the applications that need PSUADE (OUU, UQ, etc) would be able to explicitly check for PSUADE without preventing the rest of the FOQUS GUI from being loaded.

**Comment by ksbeattie** [comment]
See #1153 for an examples of this and a possible solution/improvement.

**Comment by ksbeattie** [comment]
@sotorrio1 will take a look into this

**Comment by lbianchi-lbl** [comment]
- On the opposite end of the spectrum from #1153 (i.e. a general solution requiring considerable effort that would address this issue indirectly), we could try to resolve the specific problem of the PSUADE dependency by making it a hard requirement for FOQUS, i.e. FOQUS won't be able to start if PSUADE is not found
- This would allow us to simplify (i.e. get rid of) a lot of the stateful logic of using the user-editable FOQUS settings to specify the PSUADE path (which made sense when PSUADE needed to be installed manually, but is not really needed now that `psuade-lite` is available via Conda
- The only real cons I can personally think of would be:
  - Losing the ability of using the parts of FOQUS that don't require PSUADE
  - Not being able to start FOQUS, realize that PSUADE is missing, installing PSUADE and resume the FOQUS workflow without having to restart the FOQUS application
- In summary, I think the tradeoff would be worth it and we'd resolve this issue with a much smaller effort compared to #1153 or other approaches requiring developing new code

**Comment by ksbeattie** [comment]
@boverhof will take alook at @lbianchi-lbl solution of removing the run-time checks for psuade in preference to a single check at start-up.  This way if psuade is not installed FOQUS exists with a message about needing psuade installed.

**Comment by lbianchi-lbl** [comment]
I can take a stab at this and open a draft PR.
